### ISSUE TITLE
Overabundance of .xlsx files

### ISSUE THREAD

**Issue by morph8hprom** [Hyperion101010/naukri-scraper#8]
There is a massive amount of .xlsx files present in the repository, 159 by my count.  If these are not intended to be saved, they should be included in a .gitignore.  If they are intended to be saved for any purpose, they should be relocated to a different directory so that they aren't cluttering up the main source directory.

**Comment by Hyperion101010** [comment]
Yes I agree there is a need of gitignore to only push the required stuff @morph8hprom
